db/schema.rb

```diff
@@ -85,6 +85,103 @@
 
   add_index "financial_transactions", ["ordergroup_id"], name: "index_financial_transactions_on_ordergroup_id", using: :btree
 
+  create_table "forem_categories", force: :cascade do |t|
```

Comment:
Please either include database schema changes and all migrations from the plugin (by running `rake foodsoft_forum_engine:install:migrations`), _or_ include neither. The database schema is used when creating a new database (e.g. new installs), the migrations for upgrading existing installations. Both are needed.

In this case, I'd say include neither and add instructions on running `rake foodsoft_forum_engine:install:migrations` to the plugin's README. Then when, after some user demand, this might become a default Foodsoft plugin, we add this to the standard database schema.